song_translator/track/views.py

```diff
@@ -35,13 +36,19 @@ class TrackDetail(generics.RetrieveUpdateDestroyAPIView):
 
 
 class CommentList(generics.ListCreateAPIView):
+    pagination_class = PaginationComments
+    permissions_class = [permissions.IsAuthenticatedOrReadOnly]
     queryset = Comment.objects.all()
     serializer_class = CommentSerializer
+    filter_backends = (SearchFilter, OrderingFilter)
+    search_fields = ('message', 'mark')
 
     def list(self, request, pk):
         queryset = Comment.objects.filter(track_id=pk)
-        serializer = CommentSerializer(queryset, many=True)
-        return Response(serializer.data)
+        res = self.filter_queryset(queryset)
```

Comment:
try not to include other changes not concerning the feature you implement. like your PR is about feature flagging, but you refactor some serializers and add pagination